app/lib/random_filename.rb

```diff
@@ -0,0 +1,19 @@
+class RandomFilename
+  attr_reader :file_type
+
+  def initialize(file_type)
+    @file_type = file_type
+  end
+
+  def generate
+    "#{SecureRandom.uuid}.#{file_extension(file_type)}"
+  end
+
+  private
+  def file_extension(file_type)
+    case file_type
+    when 'image/jpeg', 'image/jpg'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.